pipelines/batch/src/main/java/com/google/fhir/analytics/FhirSearchUtil.java

```diff
@@ -317,6 +332,88 @@ Bundle searchByPatientAndLastDate(
     return query.execute();
   }
 
+  /**
+   * Fetch the details of bulk export job for the given bulkExportStatusUrl
+   *
+   * @param bulkExportStatusUrl the url of the bulk export job
+   * @return BulkExportHttpResponse - the status and details of the bulk export job
+   * @throws IOException
+   */
+  public BulkExportHttpResponse fetchBulkExportHttpResponse(String bulkExportStatusUrl)
+      throws IOException {
+    IHttpResponse httpResponse = fetchUtil.fetchResponseForUrl(bulkExportStatusUrl);
+    BulkExportHttpResponse bulkExportHttpResponse = new BulkExportHttpResponse();
+    bulkExportHttpResponse.setHttpStatus(httpResponse.getStatus());
+    if (httpResponse.getHeaders("expires") != null
+        && !httpResponse.getHeaders("expires").isEmpty()) {
+      String expiresString = httpResponse.getHeaders("expires").get(0);
+      Date expires = new Date(expiresString);
+      bulkExportHttpResponse.setExpires(expires);
+    }
+    if (!CollectionUtils.isEmpty(httpResponse.getHeaders("retry-after"))) {
+      String retryHeaderString = httpResponse.getHeaders("retry-after").get(0);
+      bulkExportHttpResponse.setRetryAfter(Integer.valueOf(retryHeaderString));
+    }
+    if (!CollectionUtils.isEmpty(httpResponse.getHeaders("x-progress"))) {
+      String xProgress = httpResponse.getHeaders("x-progress").get(0);
+      bulkExportHttpResponse.setXProgress(xProgress);
```

Comment:
I actually asked about this in `BulkExportUtil`; does HAPI provide this and in what granularity, e.g., 1%, 10%, 50% increments?